docs/source/xrpl.utils.tx_parser.rst

```diff
@@ -0,0 +1,37 @@
+xrpl.utils.tx\_parser package
```

Comment:
This doc shouldn't be added - the docs for the new function will be in the `xrpl.utils` file